furusiya/model/skills/frostbomb.py

```diff
@@ -0,0 +1,16 @@
+from model.components.ai.monster import FrozenMonster
+
+
+class FrostBomb:
+    @staticmethod
+    def process(area_map, player, ai_system, conf):
+        center = conf.radius // 2
+        for x in range(player.x - center, player.x + center + 1):
+            for y in range(player.y - center, player.y + center + 1):
+                entities = (
```

Comment:
I wonder if you can use `reduce` here to iterate and converge the entries into a collection. Not sure how you're aggregating them into the set right now.